tests/pageobjects/class-details.po.ts

```diff
@@ -2,7 +2,17 @@ import { Selector } from 'testcafe';
 
 export default class ClassDetailsPage {
   _id: Selector;
+  classNameInput: Selector;
+  gradeSelect: Selector;
+  gradeSelectOption: Selector;
+  backToClassButton: Selector;
+  classRow: Selector;
   constructor() {
     this._id = Selector('[data-test-id$="class-details-page"]');
+    this.classNameInput = Selector('[data-test-id$="class-name-input"]');
```

Comment:
I would consider making those properties `static readonly` then you won't have to create an instance of the `Page` class. Not that of a difference, just can simplify a bit